site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -70,9 +70,30 @@ The specified Lambda Id must be of type `SAMLv2Reconcile`.
 
 include::_identity-provider-linking-strategy-request-parameter.adoc[]
 
+[field]#identityProvider.loginHintConfiguration.enabled# [type]#[boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.43.0#::
+When enabled and HTTP-Redirect binding are used, a login hint will be passed (when available) as a query parameter to the IdP using the configured parameter name set by the [field]#identityProvider.loginHintConfiguration.parameterName#.
+
+[field]#identityProvider.loginHintConfiguration.parameterName# [type]#[String] #Defaults to `login_hint`# [since]#Available since 1.43.0# ::
```

Comment:
You have an extra space between `#` and `::`,  I think that will break the render.  Can you confirm this renders correctly?